src/python/genny/cedar_report.py

```diff
@@ -105,7 +106,7 @@ def build_report(config):
             tags=[],
             local_path=path,
             created_at=config.created_at,
-            is_uncompressed=True
+            convert_bson_to_ftdc=True
```

Comment:
personal vendetta against booleans - can this be a `conversion_mode` or something that's an enum e.g. `bson_to_ftdc` or `no_conversion`?

(I realize it was already a boolean and this just gives it a better name so feel free to punt on this)

Reply:
I think you'd want to have poplar do this. The format of the `namedtuple`s in this file just mirror the schema defined in poplar, it seems weird to make an exception for this one field.